orgs/organization.go

```diff
@@ -15,9 +15,14 @@ import (
 	"github.com/LF-Engineering/dev-analytics-libraries/http"
 )
 
+const (
+	// TRUE ...
+	TRUE = true
```

Comment:
why this const, why not passing `true` directly

Reply:
wanted to keep the code consistent